src/tasks/03_removeFalsyValues.js

```diff
@@ -15,3 +15,12 @@
  */
 
 // <-- тут реализация removeFalsyValues
+export function removeFalsyValues(array) {
+     let mass = [];
+    array.forEach(element => {
+        if (Boolean(element) === true) {
```

Comment:
В операторе if значение в скобках должно быть boolean, ты привел его вызовом Boolean

Далее уже нет смысла выполнять операцию сравнения.

Как бы результат Boolean() это уже true или false, и получается ты дополнительно сравниваешь true === true, false === true

```suggestion
        if (Boolean(element)) {
```

Reply:
Ну и вообще, под такую задачу можно использовать функцию .filter
```
array.filter(item => Boolean(item))
```